src/app/components/side-panel.js

```diff
@@ -29,8 +31,9 @@ const css = csjs`
     cursor: pointer;
   }
   .pluginsContainer {
-    height: calc(100% - 35px);
-    overflow: auto;
+    height: 100%;
+    flex: 1;
+    padding-bottom: 40px;
```

Comment:
Why ?

Reply:
because otherwise the top is shifted when it's not an iframe